generators/generator-base.js

```diff
@@ -262,29 +270,41 @@ Generator.prototype.getAllInstalledLanguages = function () {
  * get all the languages supported by JHipster
  */
```

Comment:
Yes that would be a good idea. Not sure how to reach them all. Maybe just putting it in the release notes and emailing all the devs is good enough?